app/renderer/components/common/sortableTable.js

```diff
@@ -32,16 +33,64 @@ class SortableTable extends React.Component {
     }
     this.counter = 0
     this.sortTable = null
+    this.dimensionCount = null
   }
   componentDidMount () {
-    this.sortTable = tableSort(this.table)
+    this.sortTable = tableSort(this.table, {
+      descending: this.props.defaultHeadingSortOrder === 'desc'
+    })
     return this.sortTable
   }
   componentDidUpdate (prevProps) {
-    if (this.props.rows &&
-      (!prevProps.rows ||
-      prevProps.rows.length !== this.props.rows.length)) {
-      this.sortTable.refresh()
+    if (this.isMultiDimensioned) {
+      let count = 0
+      if (this.dimensionCount == null && prevProps.rows) {
+        for (let i = 0; i < prevProps.rows.length; i++) {
+          if (this.props.rows[i].length > 0) {
+            if (this.props.rows[i]) {
```

Comment:
Is this inner `if` necessary? the upper check for `this.props.rows[i].length > 0` wouldn't pass for falsy objects.

Reply:
good catch, not needed